readme.usage.md

```diff
@@ -100,30 +100,38 @@ use a near-unique value for each parameter to avoid collisions.
 ## Usage
 
 ```
```

Comment:
These changes are all from copying the output of `sesam -h` into this readme. Some commands have just been re-ordered but there are some new ones as well as changes to some descriptions.